site/app/views/calendar/CalendarView.php

```diff
@@ -48,9 +53,12 @@ public function showCalendar(CalendarInfo $info): string {
             "curr_month" => date("n"), // the current month
             "curr_day" => date("d"),   // the current date
             'date_format' => $this->core->getConfig()->getDateTimeFormat()->getFormat('gradeable'),
-            "gradeables_by_date" => $info->getInfoByDate(),
-            "gradeables_by_section" => $info->getInfoBySections(),
+            "gradeables_by_date" => $info->getGradeablesByDate(),
+            "gradeables_by_section" => $info->getGradeablesBySections(),
             "empty_message" => $info->getEmptyMessage(),
+            "in_course" => $in_course,
+            "items_by_date" => $info->getItemsByDate(),
+            "is_instructor" => $this->core->getUser()->getGroup() === 1
```

Comment:
Should the constant 1 be replaced with a constant value which better describes what it is?  If I recall correctly, the constant is `User::GROUP_INSTRUCTOR`.